dist/electron-cookies.js

```diff
@@ -13069,20 +13069,15 @@ const { Cookie } = tough;
 // it's possible to send these cookies to a domain someone could register after the fact, but
 // for Heap, we need a parseable URI because internally we try to determine the right level to set
 // a cookie, rather than having a known set of public suffix domains.
-const FAKE_APP_URI = 'https://yourdomain.heap/';
+const FAKE_APP_URI = 'https://' + window.location.host;  
 
 (function(document) {
-  const store = new WebStorageCookieStore(localStorage);
-
-  const cookiejar = new tough.CookieJar(store);
-  Object.defineProperty(document, "cookie", {
-    get() {
-      return cookiejar.getCookieStringSync(FAKE_APP_URI);
-    },
-    set(cookie) {
-    // loose: true lets us accept cookies with key and no value, which the
-    // original tests for this library included.
-      cookiejar.setCookieSync(Cookie.parse(cookie, {loose: true}), FAKE_APP_URI);
+const store = new WebStorageCookieStore(localStorage);
+const cookiejar = new tough.CookieJar(store);
+const cookiejar = new tough.CookieJar(store, false); //false here sets the rejectPublicSuffix flag
```

Comment:
The argument that `tough.CookieJar` is expecting is an object with the flag names and values, like:

`tough.CookieJar(store, {rejectPublicSuffix: false});` -- which is  also nice because it's self documenting, unless you want to explain that we need that flag for Cordova compatibility.